examples/tensorflow-akka/build.sbt

```diff
@@ -0,0 +1,53 @@
+//tag::docs-projectSetup-example[]
+import sbt._
+import sbt.Keys._
+
+import scalariform.formatter.preferences._
+
+lazy val tensorflowAkka =  (project in file("."))
+    .enablePlugins(CloudflowAkkaStreamsApplicationPlugin)
+    .settings(
+//end::docs-projectSetup-example[]
+      libraryDependencies ++= Seq(
+        "ch.qos.logback"         %  "logback-classic"           % "1.2.3",
+        "com.typesafe.akka"      %% "akka-http-testkit"         % "10.1.10" % "test",
+        "org.tensorflow"         % "tensorflow"                 % "1.13.1",
+        "org.tensorflow"         % "proto"                      % "1.13.1",
+        "org.scalatest"          %% "scalatest"                 % "3.0.8"  % "test"
+//tag::docs-projectName-example[]
+      ),
+      name := "tensorflow-akka",
+//end::docs-projectName-example[]
+      organization := "com.lightbend",
+
+      scalaVersion := "2.12.10",
+      crossScalaVersions := Vector(scalaVersion.value),
+      scalacOptions ++= Seq(
+        "-encoding", "UTF-8",
+        "-target:jvm-1.8",
+        "-Xlog-reflective-calls",
+        "-Xlint",
+        "-Ywarn-unused",
+        "-Ywarn-unused-import",
+        "-deprecation",
+        "-feature",
+        "-language:_",
+        "-unchecked"
+      ),
+
+      runLocalConfigFile := Some("src/main/resources/local.conf"),
+
+      scalacOptions in (Compile, console) --= Seq("-Ywarn-unused", "-Ywarn-unused-import"),
+      scalacOptions in (Test, console) := (scalacOptions in (Compile, console)).value,
+
+      scalariformPreferences := scalariformPreferences.value
```

Comment:
The Scala world seems to rapidly move to VS Code and Metals, we should use `scalafmt` instead.

Reply:
Probably best to do this in a separate PR, for all examples. I've added an issue https://github.com/lightbend/cloudflow/issues/74